src/vunnel/providers/rhel/parser.py

```diff
@@ -229,6 +233,12 @@ def _sync_cves(self, skip_if_exists=False, do_full_sync=True):  # noqa: PLR0915,
         utils.silent_remove(os.path.join(self.cve_dir_path, self.__last_synced_filename__))
         utils.silent_remove(os.path.join(self.cve_dir_path, self.__cve_download_error_filename__))
 
+        if self.fixdater:
+            if self.skip_download:
+                self.logger.warning("skip download requested, but fix date finder does not support skipping download")
```

Comment:
This would be nice to add some day, mostly so that when I'm iterating on a provider I don't need to download the fix date info every time. Definitely out of scope for this PR though :)